addons/html_editor/static/tests/link/popover.test.js

```diff
@@ -508,6 +508,23 @@ describe("Link creation", () => {
                 `<p>[<a href="https://www.test.com">Hello</a> my friend]</p>`
             );
         });
+        test("should display the metadata popover when a link element is selected", async () => {
+            await setupEditor(`<p><a href="https://www.test.com">test</a> [link]</p>`);
+            await waitFor(".o-we-toolbar");
+            await click(".o-we-toolbar .fa-link");
+            await waitFor(".o-we-linkpopover", { timeout: 1500 });
+            expect(".o-we-linkpopover input.o_we_href_input_link").toHaveCount(1);
+            const aNode = document.querySelector("a");
```

Comment:
It's nice to have a test, but since it's a refactoring and the behavior of link popover doesn't change, I doubt we actually need this test. If my first sight is right, this test passes without your current change, which means we don't need it

Reply:
The thing is that this new test could be decomposed to two existing tests:
1. `should convert all selected text to link` + selection change 
2. `link popover should have buttons for link operation when the link has href`. Here in this test, we test `expect(".o_we_url_link").toHaveCount(1);` But eventually it meant to make sure the popover is at non-editing state

I would prefer to remove the test and keep the unit tests focusing on small functionality, because we can always create a test combining a few existing tests, but it won't increase the actual coverage.

Pls feel free to let me know if I'm missing something :pray: